hack/jenkins/pipelines/hyperkube-release/Jenkinsfile

```diff
@@ -0,0 +1,30 @@
+
+podTemplate(label: 'hyperkube-build',
```

Comment:
If you're only using a single container, is it worth trying to switch back to the declarative pipeline format? Maybe it doesn't matter as much here, since we don't have critical cleanup steps that must run (like we do with running terraform destroy in the e2e jobs).